docs/book/installation.rst

```diff
@@ -74,8 +95,41 @@ browser.
 
 .. tip::
 
-    You can also use the Symfony built-in webserver by running the ``php app/console server:run --env dev``
-    command and then accessing ``http://localhost:8000``.
+    We strongly recommend using the Symfony abuilt-in webserver by running the
+    ``php app/console server:run --env dev``
+    command and then accessing ``http://localhost:8000``
+
+.. note::
+
+    The localhost's 8000 port may be already occupied by some other process.
+    If so you should try other ports, like for instance:
+    ``php app/console server:run localhost:8081 --env dev``
+
+You should be already logged in as an administrator, with the credentials you have provided during the installation process.
+Since now you can test your clean Sylius installation.
+
+Accessing the Administration Panel
+----------------------------------
+
+In order to see a fully functional Administration you will need to install `Gulp`_.
+
+**Sylius** already has a ``gulpfile.js``, therefore you just need to get Gulp using `Node.js`_.
+
+Having Node.js installed in your project directory run:
+
+.. code-block:: bash
+
+    $ npm install
```

Comment:
There is a link -> https://github.com/Sylius/Sylius/pull/5174#discussion-diff-65704525R116 here.